src/java.base/share/classes/java/util/stream/Gatherer.java

```diff
@@ -115,8 +115,8 @@
  * Gatherer<Integer, ?, String> incrementThenToString = increment.andThen(toString);
  * }
  *
- * <p>As an example, a Gatherer implementing a sequential Prefix Scan 
- * could be done the following way:
+ * <p>As an example, a Gatherer implementing a sequential Prefix Scan could
```

Comment:
@AlanBateman @PaulSandoz Somehow having a trailing space after "Scan" was preventing integration-ability of this PR. Apologies.